compiler/fir/checkers/src/org/jetbrains/kotlin/fir/analysis/checkers/FirHelpers.kt

```diff
@@ -197,6 +200,16 @@ fun CheckerContext.findClosestClassOrObject(): FirClass<*>? {
     return null
 }
 
+fun CheckerContext.findClosestFile(): FirFile? {
```

Comment:
@mglukhikh  I just found there is `CheckerContext#findClosest<T>()`. Feel free to remove this. Also noticed that, in your experimental branch, its usage is also gone?